addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -6733,6 +6773,23 @@ registry.ImageTools = ImageHandlerOption.extend({
      */
     async _writeShape(svgText) {
         const img = this._getImg();
+
+        // Apply the right animation speed if there is an element animate.
+        
+        if (this._isAnimatedShape() && !img.dataset.hoverEffect) {
+            if (!img.dataset.animationRatio) {
+                img.dataset.animationRatio = this.currentAnimationRatio || "5";
+            }
+            svgText = svgText.replace(/((\d+(\.\d+)?)|(\.\d+))s([\"\s])/gm, function(matched) {
+                const ratio = (10 ** ((img.dataset.animationRatio - 5) / 5)).toFixed(1);
+                const content = matched.slice(0, -1).replace("s", "");
+                const separator = matched.slice(-1);
+                return `${(parseFloat(content) / parseFloat(ratio)).toFixed(5)}s${separator}`;
+            });
+        } else {
+            delete img.dataset.animationRatio;
```

Comment:
Normally you shouldn't need to do this.. Why do you need to do this? This must be done once a shape has been selected depending on whether it is animated or not (See other shape options ;) )